api-reference/10 UI Components/dxOverlay/1 Configuration/wrapperAttr.md

```diff
@@ -5,7 +5,108 @@ default: {}
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes" target="_blank">global attributes</a> for the UI component's wrapper element.
+
+---
+---
+##### jQuery
+
+    $(function(){
+        $("#{widgetName}Container").dx{WidgetName}({
+            // ...
+            wrapperAttr: {
+                id: "elementId",
+                class: "class-name"
+            }
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html-->
+    <dx-{widget-name} ...
+        [wrapperAttr]="{ id: 'elementId', class: 'class-name' }">
+    </dx-{widget-name}>
+
+    <!--tab: app.component.ts -->
+    import { Component } from '@angular/core';
+    // ...
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+    export class AppComponent {
+        // ...
+    }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { Dx{WidgetName}Module } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            Dx{WidgetName}Module
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <Dx{WidgetName} ...
+            :wrapper-attr="{widgetName}Attributes">
+        </Dx{WidgetName}>
+    </template>
+
+    <script>
+    import Dx{WidgetName} from 'devextreme-vue/{widget-name}';
+
+    export default {
+        components: {
+            Dx{WidgetName}
+        },
+        data() {
+            return {
+                {widgetName}Attributes: {
+                    id: 'elementId',
+                    class: 'class-name'
+                }
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import {WidgetName} from 'devextreme-react/{widget-name}';
+
+    function App() {
+        const {widgetName}Attributes = {
```

Comment:
Declare the object outside the function or wrap it in `useMemo`: https://js.devexpress.com/Documentation/Guide/React_Components/Component_Configuration_Syntax/#Properties_of_the_Object_Type